hieradata_aws/common.yaml

```diff
@@ -354,6 +356,7 @@ gdal::repo::apt_mirror_hostname: "%{hiera('apt_mirror_hostname')}"
 gdal::repo::apt_mirror_gpg_key_fingerprint: "%{hiera('apt_mirror_fingerprint')}"
 
 govuk::apps::publisher::port: 3000
+govuk::apps::account_api::port: 3001
```

Comment:
I'm a bit suspicious of port 3001 being unused when 3000 and 3002 are both taken.  How about using the highest listed port + 1, which would be 3243?

One more thing to look forward to after replatforming: not needing to worry about unique ports.

Reply:
It sounds sensible, I'll change that.
Although, more transparency on what ports we should and shouldn't use would be useful. 😄